Library/Formula/slimerjs.rb

```diff
@@ -1,5 +1,6 @@
 class FirefoxDependency < Requirement
   fatal true
+  default_formula "xulrunner"
```

Comment:
There's already a Yosemite failure in the formula for ` xulrunner ` - I didn't know if that would be sufficient?